src/components/Main/Burger/index.js

```diff
@@ -0,0 +1,105 @@
+import styled from "styled-components";
+import backGroundImg from "../../../assets/bg.jpg";
+import Loader from "../Loader";
+function Burger({ totalPrice, ingredientAddingOrder, loading }) {
+  const positionTopBun = ingredientAddingOrder.length
+    ? 115 + 25 * ingredientAddingOrder.length
+    : 260;
+  const zIndexTopBun = ingredientAddingOrder.length + 1;
+  return (
+    <div>
+      {loading ? (
+        <Loader />
+      ) : (
+        <Div>
+          <Tittle>Burger price: {totalPrice} ₴</Tittle>
+          <BurgerWrapper>
+            <TopBun
+              src={require("../../../assets/products/top_bun.png")}
```

Comment:
I'm not a fan of using dynamic imports for images.
As for me, it's better to do something like 
```
import Image from 'YOUR_PATH/image.png';
...
<img src={Image}/>
```